hotel_roommatik/models/inherited_hotel_folio.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Jose Luis Algara (Alda hotels) <osotranquilo@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+import xml.etree.cElementTree as ET
```

Comment:
Why use xml.etree??? I don't sure, but I think that the roommatik client can read dictionary and other elements... ¿? I will confirm it with them , but I would not use this in a first step...